submissions/Liddell777/hooli-style-popup/style.css

```diff
@@ -0,0 +1,277 @@
+* {
+    padding: 0;
+    margin: 0;
+    border: 0;
+}
+
+*, *:before, *:after {
+    box-sizing: border-box;
+}
+
+body{
+    height: 100%;
+    width: 100%;
+    line-height: 1;
+    font-size: 14px;
+    overflow: hidden;
+    font-family: "Titillium Web", sans-serif;
+}
+
+input, button, textarea {
+    font-family: inherit;
+}
+
+input::-ms-clear {
+    display: none;
+}
+
+button {
+    cursor: pointer;
+}
+
+button::-moz-focus-inner {
+    padding: 0;
+    border: 0;
+}
+
+a, a:visited, a:hover  {
+    text-decoration: none;
+}
+
+ul {
+    list-style: none;
+}
+
+img {
+    vertical-align: top;
+    max-width: 100%;
+}
+main {
+    background: 
+    url(../img/bg5.gif) center no-repeat;
+    background-size: 500px;
+}
+
+
+header {
```

Comment:
Avoid styling by HTML tag.
Separate responsibilities:
HTML tag for markup
CSS class for styling
id for JS logic and linking input with label

Reply:
sorry, I didn't understand what do you mean.. 
if it about the header tag it is used for navigation or what is my mistake? thanks

Reply:
I mean use styling by HTML tag only for common styles.
If you style this component by HTML tag it can affect all elements on other pages with this tag.
In our small project, it has no affect but on a bigger project, it will.

Reply:
now I understand, I will follow it, thanks.